Traceback (most recent call last):
  File "/home/zhangjinrong/program/Torch_Template/main.py", line 84, in <module>
    arti_modle.add_file(x[0] + x[1] + x[2] + '.py')
  File "/home/zhangjinrong/anaconda3/envs/torch/lib/python3.10/site-packages/wandb/sdk/wandb_artifacts.py", line 422, in add_file
    raise ValueError("Path is not a file: %s" % local_path)
ValueError: Path is not a file: modelsMSRGCNmsrgcn.py